stockfish/models.py

```diff
@@ -102,6 +102,42 @@ def set_position(self, moves: List[str] = None) -> None:
             moves = []
         self.__put(f"position startpos moves {self.__convert_move_list_to_str(moves)}")
 
+    def get_board_visual(self) -> None:
+        """ Get a visual representation of the current board position 
+            Note: "d" is a stockfish only command
+
+        Args:
+            None
+
+        Returns:
+            None
+        """
+        self.__put("d")
+        for _ in range(18):
+            print(self.stockfish.stdout.readline(), end = " ")
+        print("\n")
+
+    def get_fen_position(self, moves: List[str] = None) -> str:
```

Comment:
Please add test for this function

Reply:
hi, I ran `$ black .` as you said and made the changes you suggested.